.github/workflows/all.yml

```
@@ -33,6 +33,12 @@ jobs:
             go version
             test -z "$(go mod vendor && git status --porcelain)"
             go mod verify
+      - name: Check code generation
+        run: |
+          go install github.com/alvaroloes/enumer@v1.1.2
+          go install github.com/mailru/easyjson/easyjson@v0.7.7
+          go generate ./...
+          exit $(git status --porcelain 2>/dev/null | wc -l)
```

```suggestion
          exit $(git status --porcelain 2>/dev/null | wc -c)
```

If you have single line without a `\n` at the end, `wc -l` will result in `0`, try `echo -n foo | wc -l`

---

btw what was the problem with `test -z`, doesn't it do basically the same thing? :confused:

---

I was researching a solution for removing `test -z` how @MStoykov requested but in the end, I lost the focus. Yep, I will bring back the `test -z` to have consistency.

---

I think this is because I asked "do we need test -z?" but I asked that as I was left with the impressions `test -z` tests whether the returned exit status is `0` which it definitely doesn't :facepalm:  and even if it did `git status --porcelain` won't exit with non zero status it will just not print some stuff on the output.

 original [comment](https://github.com/grafana/k6/commit/da37f41e438b16353c381912bb8c91fa2b75b0ee#r54025705) ... which I think is on a commit :facepalm:  oh ... how I love github and its emails ...